docs/tutorials/integrating-with-rails.md

```diff
@@ -10,91 +10,146 @@ This integration has been tested with the following:
 
 - Rails >= 6
 - Node >= 12.10
-- Webpacker >= 5
+- Yarn >= 1.22
 
 ## Instructions
 
-To get started using Shoelace with Rails, the following packages must be installed.
+When using Shoelace, there are mostly three things that need to be served to the client browser:
+
+- Javascript files for the Web Components
+- CSS files for light and dark themes (they can co-exist, but one of them is required)
+- Shoelace Icons
+
+Depending on the JS bundler you are using, you may need to do some additional configuration. However, the basic steps
+should be just about the same. Also, it is recommended to read to the [Bundling section in the Installation](/getting-started/installation?id=bundling)
+to understand how Shoalace can be set up with a JS bundler in general. In this tutorial, we will assume that your Rails
+is already set up with a JS bundler that supports importing CSS files directly (e.g. Turbopack, esbuild, Vite).
```

Comment:
Is it worth documenting importmaps `<link>` to the CDN for stylesheets?

Reply:
I have just looked at importmaps, and wow, this seems like anther giant beast. This will take a bit more time to write up a doc for it...

Reply:
I took a deep dive into the `importmaps-rails` gem and now https://toprubycompanies.info/ runs with it. Overall, it's working great. There is a couple of changes needed in the autoloader, and I will send a separate PR for that.

<img width="939" alt="importmap" src="https://user-images.githubusercontent.com/386234/235344611-b5a92cdb-a109-41bb-b383-c41045ac059f.png">